Mp: AUM & Protocol Fees (#93)
balancer-labs/docs: docs/concepts/pools/managed.md

// Get the current value of the swap fee percentage
uint256 swapFeePercentage = _managedPool.getSwapFeePercentage();
```

## Management and Protocol Fees

> Assets Under Management (AUM) fees are paid out to the `owner` for maintaining the pool. In Managed Pools, protocol fees are only taken as a percentage of AUM fees, as opposed to the standard method of being extracted from swap fees. The AUM fee percentage can be set by an `owner` but must be less than `95e16` or 95%. The protocol fee percentage is established by [ProtocolFeePercentagesProvider](https://github.com/balancer/balancer-v2-monorepo/blob/master/pkg/standalone-utils/contracts/ProtocolFeePercentagesProvider.sol). 

Reviewer: ```suggestion
Assets Under Management (AUM) fees are paid out to the `owner` for maintaining the pool. In Managed Pools, protocol fees are only taken as a percentage of AUM fees, as opposed to the standard method of being extracted from swap fees. The maximum allowable AUM fee that an `owner` can set is 95% (represented as `95e16`). The protocol fee percentage is established by [ProtocolFeePercentagesProvider](https://github.com/balancer/balancer-v2-monorepo/blob/master/pkg/standalone-utils/contracts/ProtocolFeePercentagesProvider.sol). 
```
